src/functionalTest/java/uk/gov/hmcts/reform/draftstore/data/DraftStoreDAOTest.java

```diff
@@ -97,22 +99,20 @@ public void shouldRetrieve() throws SQLException {
         dataAgent.setupDocumentForUser(USER_ID, "default", "{ \"test\":\"1234\"}");
         givenExistingDocument(ANOTHER_USER_ID, ANOTHER_PETITION);
 
-        String document = underTest.retrieve(USER_ID, "default");
+        List<Draft> drafts = underTest.readAll(USER_ID, "default");
 
-        assertThat(document).isEqualTo("{ \"test\":\"1234\"}");
+        assertThat(drafts.size()).isEqualTo(1);
+        assertThat(drafts.get(0).document).isEqualTo("{ \"test\":\"1234\"}");
         assertNoOtherUserDataHasBeenAffected();
     }
 
-    @Test(expected = NoDraftFoundException.class)
-    public void shouldThrowOnRetrieveWhenNoDocumentForUser() throws SQLException {
-        underTest.retrieve(USER_ID, "default");
-    }
```

Comment:
repo no longer throw exception, moved to controller

Reply:
I'd replace that test with one that checks if the DAO returns an empty list.

Reply:
It's also worth adding a test checking if we can retrieve multiple drafts.